kalite/topic_tools/__init__.py

```diff
@@ -459,9 +459,28 @@ def get_assessment_item_data(request, assessment_item_id=None):
         return None
 
     # TODO (rtibbles): Enable internationalization for the assessment_items.
+    logging.info('Wrapping the Assessment Item with _()')
+    lang_code = i18n.get_default_language()
+    activate(lang_code)
```

Comment:
Do we need to activate the language here?  Isn't the language supposed to be active on the back-end already?